File: Javascript/7kyu/isograms.js
//Describtion:

//My Solution:
function isIsogram(str){
    var word = str.toLowerCase();
    var wordSet = new Set(word);
    var lenWord = word.length;
    var lenWordSet = wordSet.size;

    if (lenWord === lenWordSet) {
        return true
    } else {
        return false
    }
}

//Other solutions:
function isIsogram(str){
	return new Set(str.toUpperCase()).size == str.length;
}

//Another one:
function isIsogram(str){ 
    return !/(\w).*\1/i.test(str)
  }

File: Python/7kyu/isograms.py
#Decribtion:

#Solution:
def is_isogram(string):
    string = string.lower()
    
    return len(set(string)) == len(string)
    
#Other solutions:
def is_isogram(string):
    string = string.lower()

    for letter in string:
        if string.count(letter) > 1: return False
    return True 